packages/fetch-and-compile/lib/index.ts

```diff
@@ -0,0 +1,28 @@
+import debugModule from "debug";
+const debug = debugModule("fetch-and-compile");
+import type Config from "@truffle/config";
+import * as Recognizers from "./recognizers";
+import { fetchWithRecognizer } from "./fetch";
+import type * as Types from "./types";
+
+export { fetchWithRecognizer };
+
+export async function fetchSingle(
+  address: string,
+  config: Config
+): Promise<Types.SingleResult> {
+  const recognizer = new Recognizers.SingleRecognizer(address);
+  await fetchWithRecognizer(recognizer, config);
+  return recognizer.getResult();
+}
+
+//note: this function is called primarily for its side-effects
+//(i.e. adding compilations to the debugger), NOT its return value!
+export async function fetchDebug(
```

Comment:
Eh, I guess?

Reply:
I mean, the way you have it, "debug" sounds like the object of the imperative "fetch"

Reply:
(It's worth keeping in mind that people are lazy and will stop attempting to use a library immediately instead of loading even a tiny bit of contextual shorthand into their working memory.)